drivers/fuel_gauge/sbs_gauge/sbs_gauge.c

```diff
@@ -173,6 +177,27 @@ static int sbs_gauge_get_prop(const struct device *dev, struct fuel_gauge_get_pr
 	return rc;
 }
 
+#ifdef SBS_GAUGE_CUTOFF_SUPPORT
+static int sbs_gauge_do_battery_cutoff(const struct device *dev)
+{
+	int rc;
+	const struct sbs_gauge_config *cfg = dev->config;
+
+	if (cfg->cutoff_support == false) {
+		return -ENOTSUP;
+	}
+
+	for (int i = 0; i < ARRAY_SIZE(cfg->cutoff_payload); i++) {
```

Comment:
```suggestion
	for (int i = 0; i < cfg->cutoff_payload_size; i++) {
```

Reply:
Actually `cutoff_payload_size` only exists in the emulator as a helper to prevent usages of `ARRAY_SIZE()`.

But  I realize that's a bit confusing so I'll remove the emulator's cutoff_payload_size in favor of `ARRAY_SIZE()`.

Reply:
The driver should capture a `cutoff_payload_size`.  `ARRAY_SIZE(cfg->cutoff_payload)` will always be equal to `SBS_GAUGE_CUTOFF_PAYLOAD_MAX_SIZE`.

This code will write a 0 to the cutoff register if the user only provides a single payload value.

Reply:
Oh jeez. I should've jumped off earlier looking back at that this morning. Thanks!